recipes/prance/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "prance" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/prance-{{ version }}.tar.gz
+  sha256: 86ec64378036471efdd9681648d8da6b39e5143ea2e6981b7863a33fbc75d739
+
+build:
+  entry_points:
+    - prance=prance.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - chardet >=3.0,<4
+    - python >=3.6
+    - pyyaml >=5.3,<6
+    - requests >=2.22,<3
+    - semver >=2.9,<3
+    - six >=1.14,<2
+    - click
+
+test:
+  imports:
+    - prance
+  commands:
+    - pip check
+    - prance --help
+  requires:
+    - pip
+    # optional dependency needed only for `prance --help`
+    - flex-swagger
```

Comment:
Why is this here? Wouldn;t users of the package also want to run `prance --help`?

Reply:
There is a selection of three backends that one can install. One of them is sufficient. I wouldn't know how to declare a `pkg-a || pkg-b || pkg-c` in a recipe.

Reply:
I'm not following sorry. If the help menu on one of the main commands a recipe provides doesn't run without a certain package, it seems like we should include that package. Did I interpret the comment and test correctly?

Reply:
The help commands needs one of `flex-swagger` or two other possibilities. If one of the three is installed, this is sufficient. Thus, this would work, if `flex-swagger` is not installed but one of the others. Therefore, `flex-swagger` shouldn't be a hard dependency.